hooks/node_handlers/alembicarchive_import_handler.py

```diff
@@ -0,0 +1,33 @@
+"""Hook class to handle Houdini Alembic read nodes.
+
+Designed to inherit ``base_import_handler`` i.e. define the ``hook`` attribute
+in your project configuration settings with the following inheritance:
+
+
+.. code-block:: yaml
+    :caption: env/includes/settings/tk-houdini_node_handlers.yml
+
+    node_handlers.shot_step:
+    - node_type: alembic
+      node_category: Sop
```

Comment:
```suggestion
    - node_type: alembicarchive
      node_category: Object
```

Reply:
Should these match the class attributes?

Reply:
also this